### ISSUE TITLE
CPUGadget - Virtual Memory Read Issue

### ISSUE THREAD

**Issue by Nightliss** [raiguard/ModernGadgets#98]
Hi rai,

I've got weird issue regarding virtual memory read by CPUGadget.

VM read by gadget:
![image](https://user-images.githubusercontent.com/30467858/44868638-b0b6c600-ac8b-11e8-921b-f2da5f02260a.png)

VM from CPUInfo:
![image](https://user-images.githubusercontent.com/30467858/44868689-cc21d100-ac8b-11e8-8cc7-ceaf341fd053.png)

That does not look the same. 

Rainmeter: 4.2.0 r3111 64-bit (Jul  8 2018) - English (1033)
ModernGadgets: v1.4.0-rc.3

**Comment by raiguard** [comment]
This isn't actually a bug. The virtual memory statistic in HWiNFO is "the total amount of memory available for allocation/swapping between main memory and disk" ([source](https://www.hwinfo.com/forum/Thread-Virtual-Memory-Which-one)), which includes physical RAM, the page file, and caches (I think). What the gadget is showing is the physical RAM usage, and the page file usage. These will not accurately reflect the total virtual memory usage reported by HWiNFO.

In other words, what you are looking at is not the virtual memory, but the physical memory and the page file separately.
